rush.json

```diff
@@ -1950,6 +1950,16 @@
       "projectFolder": "sdk/loadtesting/load-testing-rest",
       "versionPolicyName": "client"
     },
+    {
```

Comment:
Do you have a deprecation plan in place for the old names? https://github.com/Azure/azure-sdk-for-js/blob/4c34225491ad3e7cdbdc50f364672d3b1588351c/rush.json#L2223-L2232

And do you plan to tackle in a separate PR?

Reply:
Yes, we have a deprecation plan for the older SDKs. Will take it up in a later PR.